redbot/cogs/audio/audio.py

```diff
@@ -122,7 +122,7 @@ def __init__(self, bot):
             status=False,
             use_external_lavalink=False,
             restrict=True,
-            current_version=redbot.core.VersionInfo.from_str("3.0.0a0").to_json(),
+            current_version=redbot.core.VersionInfo.from_str(__version__).to_json(),
```

Comment:
What is this value being used for right now?

If it is or has been used for migrating data, this is not a correct change and will break anyone upgrading from the 3.0.0 -> 3.2.0 as the default here should be the lowest version, with migrations setting the version related to their migration.

Reply:
afaik this is not currently used for anything ... but @aikaterna  should be able to confirm it already existed before I touched audio..

Reply:
This is unused anywhere within red ... I've remove this default